README.md

```diff
@@ -1 +1,93 @@
-# angular-event-directives-readme
\ No newline at end of file
+# Event based Directives
+
+## Overview
+
+Now that we've talked about what a directive actually is, let's go into what an event directive actually does for us.
+
+## Objectives
+
+- Describe event based Directives
+- Use native events to bind events to a set of list items
+- Manually bind/unbind events on list items
+- The reason for Angular events
+- Use event based Directives to handle events
+- Create an event function in the Controller
+- Assign the event function to a Directive
+- Update the View after event function has been invoked
+
+## Event based directives
+
+If you've ever done a lot of JavaScript before, you would have come into contact with event listeners. These are functions that get called whenever a specific behaviour happens that we are looking out for. Either of the following might be familiar to you:
```

Comment:
really like that you show how this might look using vanilla Javascript and explain the problems with it. Can you use an event listener for click to keep it consistent with the angular example below?